### DISCUSSION TITLE
Why does crd need to be applied separately instead of being placed in the chart package？

### DISCUSSION THREAD

**Discussion by acechef** [k8up-io/k8up#778] [Q&A]
Why does crd need to be applied separately instead of being placed in the chart package？

**Comment by ccremer** [comment]
Hi.
Because the CRDs are not updated with a Helm upgrade.
If we add more fields to the specs in a release (e.g. v2.6), they won't be available if they are installed via Helm only (e.g. first installation with v2.0).

More details here: https://github.com/appuio/charts/issues/419#issuecomment-1114607107 and https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

**Reply by ccremer** [reply]
@acechef any reason for unmarking my answer? Yet you didn't leave a comment explaining why. Is something not clear?

**Reply by ccremer** [reply]
Marking as answer again and lock...
